stdlib/builtin.ncl

```diff
@@ -169,9 +180,10 @@
 
     serialize : [| `Json, `Toml, `Yaml |] -> Dyn -> String
     | doc m%"
-      Serializes the given value to the desired representation.
+      Serialize a value into the desired representation.
```

Comment:
Is serialize into more idiomatic than serializing to a format ? The latter actually sounds more natural to me, but I can't say for sure.

Reply:
To be honest, I'm not sure. I think, I'm using it similarly to "convert into". FWIW the Wikipedia article on serialization agrees with me in its first sentence 😅